src/libraries/System.Private.CoreLib/src/System/Math.cs

```diff
@@ -950,6 +950,36 @@ public static double MinMagnitude(double x, double y)
             return y;
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
Add docs?

Reply:
Yes, need to do that for this one.

Actually, @carlossanlop is the bot running, this seems to be missing that tag reminding me to add them 😄

Reply:
A bot added the "new-api-needs-doc" label already right? Or is there another bot?

Reply:
Ah it's, there, I just missed it completely. It blends in with dark theme:
![image](https://user-images.githubusercontent.com/10487869/104056792-e5b17c80-51a5-11eb-8629-00979bb0404a.png)